src/data/store/Features.js

```diff
@@ -145,6 +145,14 @@ Ext.define('GeoExt.data.store.Features', {
         me.bindLayerEvents();
     },
 
+    applyFields: function(fields) {
+        if (fields) {
+            this.setModel(
+                Ext.data.schema.Schema.lookupEntity('GeoExt.data.model.Feature')
```

Comment:
A tiny wish: make the actual model configurable. This allows subclassing.

Do it in this pr or elsewhere if you agree.